.github/workflows/deployDoc.yml

```diff
@@ -22,12 +22,18 @@ jobs:
     - name: Configure CMake
       run: cmake -B "${{github.workspace}}/build" -DHUB_BUILD_DOC=ON
       
-    - name: Build doxygen doc
-      run: cmake --build ${{github.workspace}}/build --target hub-doc-doxygen
```

Comment:
So if I understand well we do not deploy doxygen documentation anymore ?

Reply:
The doxygen documentation is also deployed by sphinx